src/cmds/qsub.c

```diff
@@ -3723,6 +3723,13 @@ process_opts(int argc, char **argv, int passet)
 		exit_qsub(1);
 	}
 #endif
+
+	if (errflg == 0 && J_opt == 0 && get_attr(attrib, ATTR_m, NULL) != NULL &&
+		strchr(get_attr(attrib, ATTR_m, NULL), 'j') != (char *)0) {
```

Comment:
Please compare the result of strchr() against NULL rather than (char *)0.

Reply:
Done